suites/squid/rados/tier-3_rados_test-stretch-mode-baremetal.yaml

```diff
@@ -0,0 +1,597 @@
+# Use cluster-conf file: conf/quincy/rados/stretch-mode-host-location-attrs.yaml
```

Comment:
Please change this to say - `conf/squid/rados/stretch-mode-host-location-attrs.yaml`

Reply:
Made the changes as per the comments.